anoncreds/tests/anoncreds_demos.rs

```diff
@@ -35,7 +34,7 @@ fn anoncreds_works_for_single_issuer_single_prover() {
 
     // Issuer creates Credential Definition
     let cred_def_parts = issuer::create_credential_definition(
-        &issuer_wallet.did,
+        "SAMPLE_ID".to_owned(),
```

Comment:
This should fail, as it is not a valid URI -- that's why I would like to keep id validation in place where we use it